lib/Installation/Partitioner/LibstorageNG/v4_3/AddLogicalVolumePage.pm

```diff
@@ -54,10 +54,11 @@ sub set_custom_size {
 
 sub select_role {
     my ($self, $role) = @_;
+    $role //= '';
     if ($role eq 'operating-system') {
         $self->{rb_system}->select();
     }
-    elsif ($role eq 'swap') {
+    elsif (($role // '') eq 'swap') {
```

Comment:
We don't need this change anymore. That was an idea of the suggestion =)

Reply:
good catch, I missed to revert that line.

Reply:
done, in follow-up ticket we should apply that as common approach, there are many string comparison leaving warnings in the log file.

Reply:
Right, we should act on those if we see such failures in the logs.